tests/test_verify_profiles.py

```
@@ -53,6 +54,43 @@ def _step(name: str) -> Any:
+        "    return orig(*a, **k)\n"
+        "sys.stdout.reconfigure = spy\n"
+        "import importlib.util\n"
+        "spec = importlib.util.spec_from_file_location(\n"
+        "    'abicheck_scripts_verify_import_check', 'scripts/verify.py'\n"
+        ")\n"
+        "mod = importlib.util.module_from_spec(spec)\n"
+        "sys.modules[spec.name] = mod\n"
+        "spec.loader.exec_module(mod)\n"
+        "print(len(calls))\n"
+    )
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)